index.html

```diff
@@ -212,13 +232,16 @@ <h2>About me</h2>
     </section>
 
     <footer class="contact-me">
-      <h2>
-        I'm always<br />
-        interested in <br />hearing about<br />
-        new projects, so <br />if you'd like to <br />chat please get<br />
+            <div>
+     <div class="last-container">
+      <div class="last-paragraph">
+        <h2>
+        I'm always
+        interested in hearing about
+        new projects, so if you'd like to chat please get
         in touch.
```

Comment:
- [ ] Kindly add space between the previous section and the contact section to match Figma's design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206476130-017df431-046b-4d18-bb3f-0b4be691d95b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206476209-83f98f25-c506-46b9-a181-d81cd73829cf.png)